src/odemis/acq/test/stream_test.py

```diff
@@ -2687,10 +2686,13 @@ def test_streak_acq_integrated_images(self):
         streaks.detStreakMode.value = True
         sems.emtDwellTime.value = 1e-06
 
+        # set a baseline, which does not effect data, but needed later to verify baseline is handled correctly
+        self.streak_ccd.updateMetadata({model.MD_BASELINE: 0})
+
         # set stream VAs
         streaks.integrationTime.value = 2  # s
         # TODO use fixed repetition value -> set ROI?
-        streaks.repetition.value = (3, 5)
+        streaks.repetition.value = (2, 2)
```

Comment:
Usually, I avoid the same repetition in X and Y in the test, as if for some reason X&Y get swapped, it's easier to detect. => 2x3?

Reply:
Agree. I had 2x3, but then this `self._roiToPhys` made it 2x2. I think, that is also why I put the TODO there at some stage. I can try another combination and see if the code keeps it, which is different in x,y.